olive/systems/azureml/aml_evaluation_runner.py

```diff
@@ -42,12 +42,22 @@ def main(raw_args=None):
 
     # load model
     model_config = get_model_config(common_args)
-    model = ModelConfig.from_json(model_config).create_model()
-
     # load metric
     metric_config = json.load(open(metric_args.metric_config))
     metric = create_metric(metric_config, metric_args)
 
+    # Replace hf config model_name with input model path to load model from input model path
+    hf_model_name = None
+    if model_config["config"].get("hf_config") and not model_config["config"]["hf_config"]["load_model_from_hub"]:
```

Comment:
If we did that loading from `model_path` in `PytorchModel` that I suggested in the other comment, I think this step could be skipped?